contracts/pools/PoolCollection.sol

```diff
@@ -207,6 +208,17 @@ contract PoolCollection is IPoolCollection, Owned, BlockNumber, Utils {
      */
     event DepositLimitUpdated(Token indexed pool, uint256 prevDepositLimit, uint256 newDepositLimit);
 
+    /**
+     * @dev triggered when a pool's trading liquidity is reduced
+     */
+    event TradingLiquidityReduced(
```

Comment:
There's already the TradingLiquidityUpdated event that must be fired when the trading liquidity is updated

Reply:
So you want to replace the emitting of this event, with emitting the `TradingLiquidityUpdated` event?

Reply:
Fixed